manim/mobject/svg/code_mobject.py

```diff
@@ -20,48 +20,131 @@
 __all__ = [
     "Code",
     "hilite_me",
-    "get_default_style",
     "insert_line_numbers",
 ]
 
 
 import html
 import os
 from ...constants import *
-from ...container import Container
-from ...mobject.geometry import RoundedRectangle
+from ...mobject.geometry import RoundedRectangle, Dot
 from ...mobject.shape_matchers import SurroundingRectangle
 from ...mobject.svg.text_mobject import Paragraph
 from ...mobject.types.vectorized_mobject import VGroup
+from pygments.lexers import guess_lexer_for_filename
 
 import re
 from pygments import highlight
 from pygments.lexers import get_lexer_by_name
 from pygments.formatters.html import HtmlFormatter
+from pygments.styles import get_all_styles
+
+"""
```

Comment:
Shouldn't this be in `Code`'s documentation?

Reply:
now I added it to the documentation but I did not, because this seems like Extra?